examples/cactus-example-carbon-accounting-backend/src/utility-emissions-channel/typescript/src/index.ts

```diff
@@ -580,16 +580,18 @@ class EmissionsChaincode {
   }
 }
 
-const ccServerOpt: ChaincodeServerOpts = {
-  ccid: process.env.CHAINCODE_CCID,
-  address: process.env.CHAINCODE_ADDRESS,
-  tlsProps: null,
-};
+// const ccServerOpt: ChaincodeServerOpts = {
```

Comment:
@Zzocker It's okay to just delete code that you'd want to comment out like this (it's in the git history anyway it can be retrieved at will anytime in the future if needed)